packages/instant-apps-components/src/components/instant-apps-social-share/instant-apps-social-share.test.ts

```diff
@@ -1,319 +1,253 @@
-    expect(element).toBeTruthy();
-    expect(element?.shortenShareUrl).toBe(true);
-
-    socialShare.shortenShareUrl = false;
-    expect(element?.shortenShareUrl).toBe(false);
-  });
-  test('socialMedia', () => {
-    const element = document.querySelector('instant-apps-social-share');
-    expect(element).toBeTruthy();
-    expect(element?.socialMedia).toBe(true);
-
-    socialShare.socialMedia = false;
-    expect(element?.socialMedia).toBe(false);
-  });
-  test('shareIconsLayout', () => {
-    const element = document.querySelector('instant-apps-social-share');
-    expect(element).toBeTruthy();
-    expect(element?.shareIconsLayout).toBe('vertical');
-
-    socialShare.shareIconsLayout = 'horizontal';
-    expect(element?.shareIconsLayout).toBe('horizontal');
-  });
-  test('scale', () => {
-    const element = document.querySelector('instant-apps-social-share');
-    expect(element).toBeTruthy();
-    expect(element?.scale).toBe('m');
-
-    socialShare.scale = 'l';
-    expect(element?.scale).toBe('l');
-  });
-  test('successMessage', () => {
-    const element = document.querySelector('instant-apps-social-share');
-    expect(element).toBeTruthy();
-    expect(element?.successMessage).toBe('');
-
-    socialShare.successMessage = 'Your awsome map has been copied to clipboard';
-    expect(element?.successMessage).toBe('Your awsome map has been copied to clipboard');
-  });
-
-  test('defaultUrlParams', () => {
-    const element = document.querySelector('instant-apps-social-share');
-    expect(element).toBeTruthy();
-    expect(element?.defaultUrlParams).toBe(null);
-    const shareParams = {
-      center: true,
-      level: true,
-      viewpoint: true,
-      selectedFeature: false,
-      hiddenLayers: true,
-    };
-    socialShare.defaultUrlParams = shareParams;
-
-    expect(element?.defaultUrlParams).toBe(shareParams);
-
-    expect(element?.defaultUrlParams).toHaveProperty('center', shareParams['center']);
-    expect(element?.defaultUrlParams).toHaveProperty('level', shareParams['level']);
-    expect(element?.defaultUrlParams).toHaveProperty('viewpoint', shareParams['viewpoint']);
-    expect(element?.defaultUrlParams).toHaveProperty('selectedFeature', shareParams['selectedFeature']);
-    expect(element?.defaultUrlParams).toHaveProperty('hiddenLayers', shareParams['hiddenLayers']);
+import SocialShare_T9n from '../../assets/t9n/instant-apps-social-share/resources.json';
+
+describe('social share', async () => {
+  const shareParams = {
+    center: true,
+    level: true,
+    viewpoint: true,
+    selectedFeature: false,
+    hiddenLayers: true,
+  };
+  const { center, level, viewpoint, selectedFeature, hiddenLayers } = shareParams;
+
+  type cases = [string, () => HTMLInstantAppsSocialShareElement];
+  type allTests = cases[];
+
+  const testCases: allTests = [
+    [
+      'popover',
+      function createPopover() {
+        let ss = document.createElement('instant-apps-social-share');
```

Comment:
What does `ss` stand for? Can you make this const name more descriptive to what it is?